module/documents/actor/actor.mjs

```diff
@@ -1131,14 +1133,18 @@ export default class Actor5e extends SystemDocumentMixin(Actor) {
       ? game.actors.get(this.flags.dnd5e?.originalActor) : null;
     const buildConfig = this._buildSkillToolConfig.bind(this, type, hostActor);
     const doubleProf = !!relevant?.prof.hasProficiency && !!alternate?.prof.hasProficiency;
+    const pace = MovementField.getTravelPaceMode(config.pace, config.skill);
 
     const { advantage, disadvantage } = AdvantageModeField.combineFields(this.system, [
       `abilities.${abilityId}.check.roll.mode`,
       `${type}s.${type === "skill" ? config.skill : config.tool}.roll.mode`
-    ]);
+    ], {
+      advantages: { count: Number(doubleProf) + Number(pace.advantage) },
+      disadvantages: { count: Number(pace.disadvantage) }
+    });
```

Comment:
Fixes an issue where advantage granted by the rule for skill + tool prof would ignore configured advantage suppression or override.